### ISSUE TITLE
New release 

### ISSUE THREAD

**Issue by mnightingale** [SocialiteProviders/Microsoft-Graph#5]
The last tag is from 2 years ago, there have been a few commits since.
I needed the tenant changes and had to require the master branch for now.

Thanks

**Comment by mstralka** [comment]
I agree - we also need the tenant changes and don't want to use the master branch if we can help it.

**Comment by wells** [comment]
I agree. The setTenantId() is a necessary or at least ideal function for this provider. I'll use master for now, but I don't like doing this.

**Comment by harrygulliford** [comment]
2.1 has been tagged with these changed
